example/particles/particles.cpp

```diff
@@ -64,6 +75,7 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   Metadata swarm_metadata;
   pkg->AddSwarm(swarm_name, swarm_metadata);
   Metadata real_swarmvalue_metadata({Metadata::Real});
+  pkg->AddSwarmValue("t", swarm_name, real_swarmvalue_metadata);
```

Comment:
I think this will typically be used for time-dependent particle methods. Here in the transport loop we keep pushing each particle until `while (t(n) < t0 + dt)` completes. You *could* do this with just a local variable `t`, except that we need to communicate `t` along with the rest of the particle state over MPI.